1/index.html

```diff
@@ -13,6 +13,7 @@
     </nav>
```

Comment:
IMO: the aria-current="page" attribute should be moved to the "/" page and removed from this page.

Reply:
sorry. Could you please tell me the details?

Reply:
"aria-current="page"" has been deleted.

Reply:
Could you include the commit hash in your comment from next time?
![image](https://github.com/user-attachments/assets/1a8ba56a-066c-4a37-bc9e-176d1173700c)

Reply:
I understand.
I'll try adding it as a test.
6d36e974983b2a07934f7495d87f1dab36b9d363